app/routes/sitemap.js

```diff
@@ -0,0 +1,23 @@
+import express from 'express'
+import fs from 'fs'
+import path from 'path'
+import getSitemap from '../tools/getSitemap'
+
+const router = express.Router()
```

Comment:
can't import express or create a router here. 
just export a function that this:
```
export default function sitemap(){
    this.app.get('/sitemap.xml',(req,res) => {
         ...
     })
)
```
look at rediret-list.js as an example - but don't make changes to it.